lib/wovnrb.rb

```diff
@@ -51,7 +51,7 @@ def call(env)
       status, res_headers, body = @app.call(headers.request_out)
```

Comment:
(Not related to the code. Just starting a conversation).
Do we also need to support API, e.g.:
https://github.com/WOVNio/WOVN.php/blob/639fd95d248edf03fa56cae4c3c3c85628b2a4db/src/wovnio/wovnphp/API.php#L75-L77

Reply:
yes. our translation API needs to know about the custom domain config in order to modify links the same way. it's all the way down to the widget.

Reply:
Fixed in 231d01b9b0afd4866a6158435233b965b9e79951.